tutormfe_extensions/plugin.py

```diff
@@ -6,14 +6,40 @@
 
 import click
 import pkg_resources
+from tutor import config as tutor_config
 from tutor import hooks
+from tutormfe.hooks import MFE_APPS
+
 
 from .__about__ import __version__
 
 ########################################
 # CONFIGURATION
 ########################################
 
+
+@MFE_APPS.add()
+def _remove_some_my_mfe(mfes):
+    config = tutor_config.load('.')
+    for setting in config:
+        if setting.startswith('MFE_') and setting.endswith('_MFE_APP') and config[setting] is None:
+            name = setting.replace('_MFE_APP', '').replace('MFE_', '').replace('_', '-').lower()
+            mfes.pop(name)
+    return mfes
+
+@MFE_APPS.add()
+def _add_my_mfe(mfes):
+    config = tutor_config.load('.')
+    for setting in config:
+        if setting.startswith('MFE_') and setting.endswith('_MFE_APP') and config[setting] is not None:
+            mfes[config[setting]["name"]] = {
+                "repository": config[setting]['repository'],
+                "port": config[setting]["port"],
+                "version": config[setting]["version"]
+            }
+
+    return mfes
```

Comment:
I think we can consolidate this in a single function:

I tried something like this:

```python
    for setting, value in config.items():
        if setting not in CORE_MFES_CONFIG:
            continue

        mfe_config = CORE_MFES_CONFIG[setting]

        if mfe_config is None:
            mfe_list.pop(mfe_config["name"])
            continue

        mfe_list[mfe_config["name"]] = {
            "repository": mfe_config["repository"],
            "port": mfe_config["port"],
            "version": mfe_config["version"],
        }
    return mfe_list
```

where `CORE_MFES_CONFIG` is the dict with `MFE_*_MFE_APP` we have below.